### DISCUSSION TITLE
can you use issue forms?

### DISCUSSION THREAD

**Discussion by Croworbit** [lightspark/lightspark#1144] [Ideas]
do you think you can use issue forms instead of the markdown thing for new issues? https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
I feel like the current way is just too cluttered and kind of gives me a headache

**Comment by dbluelle** [comment]
I'm for it, especially because the current form adds misleading "x of y tasks" entries into the issue overview...
If somebody can make a PR with a good template, I will merge that.
